### PR TITLE
Impl user timeline

### PR DESCRIPTION
**Pull request by mmmommm**
Add user timeline APIs

related #10

### CODE REVIEW

File: tweet.go [hunk 1]
```diff
@@ -173,6 +173,58 @@ type TweetIncludes struct {
...
+	if len(t.TweetFields) > 0 {
+		q.Add("tweet.fields", strings.Join(tweetFieldsToString(t.TweetFields), ","))
+	}
+	if len(t.UserFields) > 0 {
+		q.Add("user.fields", strings.Join(userFieldsToString(t.UserFields), ","))
+	}
+	if len(q) > 0 {
+		req.URL.RawQuery = q.Encode()
+	}
+}
```
Comment (sivchari) [inline tweet.go:220]: This method is already defined at tweet_option.go
If you wanna define option's method, please write at it.

File: tweet.go [hunk 2]
```diff
@@ -173,6 +173,58 @@ type TweetIncludes struct {
...
+
+func tweetFieldsToString(tfs []TweetField) []string {
+	slice := make([]string, len(tfs))
+	for i, tf := range tfs {
+		slice[i] = string(tf)
+	}
+	return slice
```
Comment (sivchari) [inline tweet.go:227]: Same reason as above.

File: user.go [hunk 3]
```diff
@@ -75,9 +75,9 @@ type UserHashtag struct {
 }
 
 type UserMention struct {
-	Start int    `json:"start"`
-	End   int    `json:"end"`
-	Tag   string `json:"tag"`
+	Start    int    `json:"start"`
+	End      int    `json:"end"`
+	UserName string `json:"username"`
```
Comment (sivchari) [inline user.go:80]: Thanks a lot.

**Review by sivchari — CHANGES_REQUESTED** [review]
Please change.

File: tweet_option.go [hunk 4]
```diff
@@ -108,3 +108,123 @@ func (t TweetSearchOption) addQuery(req *http.Request) {
...
 }
+
+type UserTweetTimelineOpts struct {
+	Expansions      []Expansion
+	MediaFields     []MediaField
+	PlaceFields     []PlaceField
+	PollFields      []PollField
+	TweetFields     []TweetField
+	UserFields      []UserField
+	Excludes        []Exclude
```
Comment (mmmommm) [inline tweet_option.go:119]: スクリーンショット 2021-10-04 21 29 44

@sivchari 
From twitter api documentation,`Exclude` is correct for this field name, but the array is the correct one to put here, what do you think it should be called?

Comment (sivchari) [reply tweet_option.go:119]: This field can be one or two, so I want to name the field Exclude.

**Comment by sivchari** [comment]
Good work, thanks !

**Comment by sivchari** [comment]
Please rebase 👍  I fixed workflow.
If you rebase and push, reviewdog annotates your syntax error.

**Comment by sivchari** [comment]
please check file changes.

**Review by sivchari — APPROVED** [review]

**Pull request merged**